websites/L/LofiHaven/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.14",
+  "apiVersion": 2,
+  "author": {
+    "name": "horizov",
+    "id": "877848431983493132"
+  },
+  "service": "LofiHaven",
+
+  "description": {
+    "en": "Your cozy digital hideaway for focus, study, and relaxation."
+  },
+
+  "url": "lofihaven.xyz",
+  "version": "1.0.0",
+  "logo": "https://cynmqjmtk2ojy1db.public.blob.vercel-storage.com/lofihaven_logo_512x512-pwsoTs1SLgmnBZN0E3iaRcozmXgWFn.png",
+  "thumbnail": "https://cynmqjmtk2ojy1db.public.blob.vercel-storage.com/lofihaven_logo_512x512-pwsoTs1SLgmnBZN0E3iaRcozmXgWFn.png",
```

Comment:
Thumbnail should be a banner, promotional image, or screenshot of the website homepage. (doesn't have to be 512x512)